BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/DeleteAccount/DeleteAccountProcessor.swift

```diff
@@ -125,4 +127,15 @@ final class DeleteAccountProcessor: StateProcessor<DeleteAccountState, DeleteAcc
             services.errorReporter.log(error: error)
         }
     }
+
+    /// Load any initial data for the view.
+    private func loadData() async {
+        do {
+            state.shouldPreventUserFromDeletingAccount =
+                try await services.authRepository.isUserManagedByOrganization()
+        } catch {
+            coordinator.showAlert(.networkResponseError(error))
```

Comment:
🤔 If an error is thrown here then `state.shouldPreventUserFromDeletingAccount` will remain `false` and after the user taps ok then the user will be able to delete their account. Shouldn't we close the view after the user cancels the error alert?